test/Hamiltonians.jl

```diff
@@ -71,15 +65,12 @@ end
         FroehlichPolaron(OccupationNumberFS(1, 1, 1)),
         FroehlichPolaron(OccupationNumberFS(1, 1, 1); momentum_cutoff=10.0),
         momentum(HubbardMom1D(BoseFS(0, 1, 5, 1, 0))),
-        Rimu.FirstOrderTransitionOperator(HubbardRealSpace(BoseFS(1,1,1,1)), -5.0, 0.01),
-        HubbardReal1D(BoseFS(2,0,0); u=1.0im) * ExtendedHubbardReal1D(BoseFS(2,0,0))
-    ]
-        test_hamiltonian_interface(H)
-        # Check that the result of show can be pasted into the REPL. Does not work with
-        # GuidingVectorSampling because it includes a DVec.
-        if !(H isa GuidingVectorSampling)
-            @test eval(Meta.parse(repr(H))) == H
-        end
```

Comment:
Looks like your PR was made to an old version of Rimu. Please merge develop into your branch.